doc/versionHistory.rst

```diff
@@ -2,6 +2,12 @@
 Version History
 ===============
 
+v0.13.0
+-------
+
+* Remove compatibility with xml<19.
+* Update to ts-pre-commit-config 0.6.
```

Comment:
0.6.1 based on Wouter's message in slack.

Reply:
I made it compatible with the 0.6.0 as well...